src/main/java/frc/robot/subsystems/swerve/ModuleIOTalonFX.java

```diff
@@ -197,12 +201,15 @@ public void updateInputs(ModuleIOInputs inputs) {
     inputs.turnAppliedVolts = turnAppliedVolts.getValueAsDouble();
     inputs.turnCurrentAmps = new double[] {turnCurrent.getValueAsDouble()};
 
+    inputs.odometryTimestamps =
+        timestampQueue.stream().mapToDouble((Double value) -> value).toArray();
     inputs.odometryDrivePositionsMeters =
-        drivePositionQueue.stream().mapToDouble(Units::rotationsToRadians).toArray();
+        drivePositionQueue.stream().mapToDouble((Double value) -> value).toArray();
```

Comment:
Had a little more time to look at this - the issue is that `odometryDrivePositionMeters` is a `double[]` vs the `Double[]` you'd get out of just calling `stream().toArray()`. What you've got is *ok* and works because `Double` is implicitly unboxed to `double`. But it looks a little funny because you're passing an identity function to map. You might prefer:

```java
inputs.odometryDrivePositionsMeters = drivePositionQueue.stream().mapToDouble(Double::doubleValue).toArray();
```

It's a little more explicit (converting Double -> double).